app/controllers/organizations_controller.rb

```diff
@@ -1,14 +1,39 @@
 class OrganizationsController < ApplicationController
+  before_filter :authenticate_user!, only: [:new, :create]
+
   def new
     @organization = Organization.new
   end
 
   def create
-    @organization = Organization.new
-
+    @organization = Organization.new(organization_params)
     if @organization.save
-      Organizer.new(user_id: current_user.id, organization_id: @organization.id, role: "owner")
+      Organizer.create!(user: current_user, organization: @organization)
+      flash[:notice] = "Your organization has been created."
+      redirect_to organization_path(@organization)
+    else
+      flash[:errors] = @organization.errors.full_messages.join(", ")
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.